src/test/java/org/gridsuite/modification/server/modifications/DeleteAttachingLineTest.java

```diff
@@ -113,4 +115,18 @@ public void createWithNoAttachmentPointTest() {
                     content().string(new NetworkModificationException(DELETE_ATTACHING_LINE_ERROR, "Unable to find the attachment point and the tapped voltage level from lines l1, l3 and l1").getMessage())
             );
     }
+
+    @SneakyThrows
+    @Test
+    public void createNewLineWithExistingIdTest() {
+        // try to create an already existing line
+        DeleteAttachingLineInfos deleteAttachingLineInfos = (DeleteAttachingLineInfos) buildModification();
+        deleteAttachingLineInfos.setReplacingLine1Id("l2");
+        String lineAttachToAbsentLineJson = mapper.writeValueAsString(deleteAttachingLineInfos);
+        mockMvc.perform(post(getNetworkModificationUri()).content(lineAttachToAbsentLineJson).contentType(MediaType.APPLICATION_JSON))
```

Comment:
Need really to use mockMVC
`modification.toModification().control(getNetwork());`

Reply:
yes but performing the post call prove that the control/check is well integrated. I prefer this way.